parsl/executors/taskvine/utils.py

```diff
@@ -69,12 +69,14 @@ class ParslFileToVine:
     """
     def __init__(self,
                  parsl_name: str,   # name of file
+                 netloc: str,       # name of file if url
```

Comment:
so why can't you use parsl_name for the name of file like the documentation one line before says? is it parsl_name is the parsl-supplied URL when the file is a URL?